plugin/evm/vm.go

```diff
@@ -310,6 +313,10 @@ func (vm *VM) Initialize(
 
 	vm.ethConfig = ethconfig.NewDefaultConfig()
 	vm.ethConfig.Genesis = g
+	// NetworkID of EVM is equal to the ChainID
+	// But this is different than Avalanche's NetworkID
+	// which represents the Avalanche network the EVM is running on
+	// like Fuji, Mainnet, Local, etc.
```

Comment:
what's the reason for this comment?

the networkID is not necessarily the same as the chainID in the EVM, but some tooling incorrectly assumes it is and adds incorrect checks to verify this which caused us to follow this convention so that the tooling would work correctly 😅

Could we either remove this comment or clarify this point?

Reply:
changed it